Core/Net/NetFileCache.cs

```diff
@@ -295,7 +295,11 @@ public void EnforceSizeLimit(long bytes, Registry registry)
                 KspVersionCriteria aggregateCriteria = manager?.Instances.Values
                     .Where(ksp => ksp.Valid)
                     .Select(ksp => ksp.VersionCriteria())
-                    .Aggregate((a, b) => a.Union(b));
+                    .Aggregate(
+                        manager?.CurrentInstance?.VersionCriteria()
+                            ?? new KspVersionCriteria(null),
+                        (a, b) => a.Union(b)
```

Comment:
Can you change `a` and `b` to more descriptive names, like `combinedCriteria` and `nextCriteria` or so?
Helps someone like me who didn't know what `Aggregate` does until five minutes ago understanding the code.

Reply:
Done. (It's just a poorly named `fold`.)